api/app/GraphQL/Validators/PoolIsCompleteValidator.php

```diff
@@ -79,6 +81,8 @@ public function messages(): array
             'key_tasks.fr.required' => 'FrenchWorkTasksRequired',
             'your_impact.en.required' => 'EnglishYourImpactRequired',
             'your_impact.fr.required' => 'FrenchYourImpactRequired',
+            'special_note.en.required' => 'EnglishYourImpactRequired',
+            'special_note.fr.required' => 'FrenchYourImpactRequired',
```

Comment:
Little copypaste leftover, I think for the `special_note.en` you want to use the added `EnglishSpecialNoteRequired` and the same with French.

Reply:
Nice catch! Fixed by [8d10095](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8006/commits/8d100950636f94721ab3e72fa481537a1808de75)